containers/README.md

```diff
@@ -476,32 +480,36 @@ exit
 ```
 
 Back in the previous ```Powershell```:
+Start up the container with this command
```

Comment:
best to still note that it is the entire application (i.e. it could be more than one container).

Reply:
see above